### ISSUE TITLE
Add shotcut method to DiscordWebhookChannel for editing messages

### ISSUE THREAD

**Issue by Jdance-Media** [ShimmyMySherbet/DiscordWebhooks#10]
You have it so the method to edit a message is in `PostedWebhookMessage`. What if I'm trying to edit a message after the plugin restarts? I don't have that particular object stored anymore. There is also no proper constructor for `PostedWebhookMessage`.

Of course I can use `DiscordWebhookService`, but that somewhat defeats the point in the whole `DiscordWebhookChannel` object and all.

**Comment by ShimmyMySherbet** [comment]
Are you asking for a shortcut method be added to the `DiscordWebhookChannel` type?
I'm not sure if I'm missing something, but I really do not see how a slight inconvenience like that makes it 'seriously flawed'.

**Comment by Jdance-Media** [comment]
Yeah, I eventually figured out that it was not seriously flawed. But I
still think that a shortcut method would be nice considering the example
shows no usage of the service class.
